cosign.key

```diff
@@ -0,0 +1,11 @@
+-----BEGIN ENCRYPTED COSIGN PRIVATE KEY-----
```

Comment:
Should we publish the private key?

Reply:
We have to, for gorelease see and use it. Note my comment https://github.com/epinio/epinio/issues/1011#issuecomment-1321814779
On the brighter side, this private key is password protected.

Reply:
can't it be passed via an env value somehow? This seems weird. I suspect that the linked goreleaser docs assumes that the goreleaser job is private and not public.